Bisection_Method/Bisection_Method.cpp

```diff
@@ -0,0 +1,105 @@
+#include<iostream>
+#include<cmath>
+using namespace std;
+// approximation limit
+#define error 0.0001
+
+//function of the polynomial is inputted here say f(x) = x*x - 2*x + 1
+float polynomial(float x)
+{
+    return x * x - 4 * x + 3;
+}
+
+float mid(float q, float w)
+{
+    return (q + w)/2;
+}
+
+int main()
+{
+    int i;
+    float lower_bound,upper_bound,a,b;
+    cout<<"For the interval (a,b) where the root may lie : "<<endl;
+    cout<<"Enter the value of a :- "<<endl;
+    cin>>lower_bound;
+    a = lower_bound;
+    cout<<"Enter the value of b :- "<<endl;
+    cin>>upper_bound;
+    b = upper_bound;
+    if(polynomial(lower_bound) * polynomial(upper_bound) > 0)
+    {
+        cout<<"Invalid interval"<<endl;
+    }
+    else
+    {   // repeat until the required accuracy is obtained
+        while(abs(polynomial(mid(lower_bound, upper_bound))) > error)
+        {
+            cout<<"\nIteration_no = "<<i<<" ,First_no = "<<lower_bound<<" ,Second_no = "<<upper_bound<<" middle = "<<mid(lower_bound, upper_bound)<<" ,f(middle) = "<<polynomial(mid(lower_bound, upper_bound))<<" ,f(first_no)*f(middle) = "<<polynomial(lower_bound)*polynomial(mid(lower_bound, upper_bound))<<endl;
```

Comment:
break this into two line. one line should contain maximum 80 characters 
keep space around << and >>